src/avt/Plotter/avtMapper.C

```diff
@@ -123,6 +123,92 @@ avtMapper::SetUpDrawable()
 }
 
 
+// ****************************************************************************
+//  Method: avtMapper::GetVarRange
+//
+//  Purpose:
+//      Gets the range of the variable. (Artificial limits ignored).
+//
+//  Arguments:
+//      rmin          The minimum in the range.
+//      rmax          The maximum in the range.
+//
+//  Returns:    True if the extents were found, false otherwise.
+//
+//  Programmer: Kathleen Bonnell 
+//  Creation:   April 2, 2001 
+//
+//  Modifications:
+//
+//    Hank Childs, Sat Apr  7 21:18:53 PDT 2001
+//    Use GetDataRange to account for missing data.
+//
+//    Hank Childs, Fri Apr 20 14:00:26 PDT 2001
+//    Use base class method because GetDataRange was removed.
+//
+//    Kathleen Bonnell, Tue Jun 11 17:13:17 PDT 2002  
+//    Ensure rmax is set if mappers == null (rmin was being set twice). 
+//
+//    Hank Childs, Thu Oct 10 08:25:18 PDT 2002
+//    Added a return value.
+//
+// ****************************************************************************
+
+bool
+avtMapper::GetVarRange(double &rmin, double &rmax)
+{
+    if (mappers == NULL)
+    {
+        //
+        // We have been asked for the range before the input has been set.
+        //
+        rmin = 0.;
+        rmax = 1.;
+        return false;
+    }
+
+    bool rv = avtMapper::GetRange(rmin, rmax);
```

Comment:
Is  the '::' usage necessary? The function is defined in avtMapperBase, so should be directly visible to this class.  If its needed, it should be avtMapperBase::GetRange.
Same with its usage in GetCurrentVarRange.

Reply:
It is necessary since `avtVariableMapper` inherits from `avtMapper` and it has a `GetRange` that overrides the one in the base class. Changing the `avtMapperBase::GetRange` works, so I'll make the change.